docs/en_us/platform_api/source/course_structure/overview.rst

```diff
@@ -0,0 +1,36 @@
+.. _EdX Platform Course Structure API Overview:
+
+################################################
+EdX Platform Course Structure API Overview
+################################################
+
+Use the edX Platform Course Structure API to view course details, including the
+blocks in the course and the course grading policy.
+
+The edX Platform Course Structure API uses Representational State Transfer
+(REST) design principles and supports the JavaScript Object Notation (JSON)
+data-interchange format. The REST API is simple and lightweight.
```

Comment:
1. Every developer coming to these pages should know what REST and JSON are. It's unnecessary and distracting to spell it out long form all the time, just like we wouldn't say "application programming interface (API)".

2. This paragraph doesn't seem necessary. All of our APIs are REST APIs using JSON. We can state that once in the top level platform APIs overview. It's likely that people will just assume that's what we're using REST+JSON anyway, and it's immediately clear from any example or detailed docs at the endpoint level as well. The bit about "simple and lightweight" is debatable and I'd rather we just removed it entirely.

I'm citing this one example, but we do this same thing in a few other places in this PR.